packages/@divvi/mobile/src/tokens/AssetTabBar.test.tsx

```diff
@@ -45,19 +48,76 @@ describe('AssetTabBar', () => {
-  ])('selecting tab $tab fires analytics events and invokes on change', ({ tab, event }) => {
+  it('does not render collections if disableNfts is set', () => {
+    jest
+      .mocked(getAppConfig)
+      .mockReturnValue({ ...mockAppConfig, experimental: { disableNfts: true } })
     const { getAllByTestId } = render(
-      <AssetTabBar activeTab={AssetTabType.Tokens} onChange={onChange} displayPositions={true} />
+      <AssetTabBar activeTab={AssetTabType.Positions} onChange={onChange} displayPositions={true} />
     )
 
-    fireEvent.press(getAllByTestId('Assets/TabBarItem')[tab])
-    expect(AppAnalytics.track).toHaveBeenCalledTimes(1)
-    expect(AppAnalytics.track).toHaveBeenCalledWith(event)
-    expect(onChange).toHaveBeenCalledTimes(1)
-    expect(onChange).toHaveBeenCalledWith(tab)
+    const tabItems = getAllByTestId('Assets/TabBarItem')
+    expect(tabItems).toHaveLength(2)
+    expect(tabItems[0]).toHaveTextContent('tokens')
+    expect(tabItems[0].children[0]).toHaveStyle({ color: Colors.contentSecondary })
+    expect(tabItems[1]).toHaveTextContent('dappPositions')
+    expect(tabItems[1].children[0]).toHaveStyle({ color: Colors.contentPrimary })
+  })
+
+  it('returns null if collects and positions are disabled', () => {
+    jest
+      .mocked(getAppConfig)
+      .mockReturnValue({ ...mockAppConfig, experimental: { disableNfts: true } })
+    const { toJSON } = render(
+      <AssetTabBar
+        activeTab={AssetTabType.Positions}
+        onChange={onChange}
+        displayPositions={false}
+      />
+    )
+    expect(toJSON()).toBeNull()
+  })
+
+  describe.each([
+    { testName: 'all tabs enabled', positions: true, collectibles: true },
+    { testName: 'collectibles disabled', positions: true, collectibles: false },
+    { testName: 'positions disabled', positions: false, collectibles: true },
+  ])('$testName', ({ positions, collectibles }) => {
+    const cases = [{ tab: AssetTabType.Tokens, event: AssetsEvents.view_wallet_assets, index: 0 }]
+    if (collectibles) {
+      cases.push({
+        tab: AssetTabType.Collectibles,
+        event: AssetsEvents.view_collectibles,
+        index: 1,
+      })
+    }
+    if (positions) {
+      cases.push({
+        tab: AssetTabType.Positions,
+        event: AssetsEvents.view_dapp_positions,
+        index: collectibles ? 2 : 1,
+      })
+    }
+    it.each(cases)(
+      'selecting tab $tab fires analytics events and invokes on change',
+      ({ tab, event, index }) => {
+        jest
+          .mocked(getAppConfig)
+          .mockReturnValue({ ...mockAppConfig, experimental: { disableNfts: !collectibles } })
+        const { getAllByTestId } = render(
+          <AssetTabBar
+            activeTab={AssetTabType.Tokens}
+            onChange={onChange}
+            displayPositions={positions}
+          />
+        )
+
+        fireEvent.press(getAllByTestId('Assets/TabBarItem')[index])
```

Comment:
nit: instead of setting an an index in the object can we find the appropriate index?

```suggestion
    const cases = [{ tab: AssetTabType.Tokens, event: AssetsEvents.view_wallet_assets }]
    if (collectibles) {
      cases.push({
        tab: AssetTabType.Collectibles,
        event: AssetsEvents.view_collectibles,
      })
    }
    if (positions) {
      cases.push({
        tab: AssetTabType.Positions,
        event: AssetsEvents.view_dapp_positions,
      })
    }

    it.each(cases)(
      'selecting tab $tab fires analytics events and invokes on change',
      ({ tab, event }) => {
        jest
          .mocked(getAppConfig)
          .mockReturnValue({ ...mockAppConfig, experimental: { disableNfts: !collectibles } })

        const { getAllByTestId } = render(
          <AssetTabBar
            activeTab={AssetTabType.Tokens}
            onChange={onChange}
            displayPositions={positions}
          />
        )

        const tabs = getAllByTestId('Assets/TabBarItem')
        const index = cases.findIndex((c) => c.tab === tab)

        fireEvent.press(tabs[index])
```